packages/frontend/src/pages/Templates/index.tsx

```diff
@@ -42,16 +41,19 @@ export default function Templates(): JSX.Element {
           </Text>
         </Flex>
 
-        {loading ? (
-          <Center my={12}>
-            <PrimarySpinner fontSize="4xl" />
+        {/* Sanity check if our templates file is missing */}
+        {!templates || templates.length === 0 ? (
+          <Center my={24}>
+            <Text>
+              There are no templates now, please contact support@plumber.gov.sg
+            </Text>
           </Center>
```

Comment:
!templates could also mean it's still loading

Reply:
actually i dont need this already